fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpIdentifierConverter.java

```diff
@@ -80,7 +80,7 @@ public String toInternalId(final String httpUri, final boolean encoded) {
 
             return FEDORA_ID_PREFIX + fedoraId;
         }
-        throw new IllegalArgumentException("Cannot translate NULL path");
+        throw new IllegalArgumentException(String.format("Cannot translate NULL path extracted from URI %s", httpUri));
```

Comment:
Any reason to use String.format here instead of a simple `+` concatenation?

Reply:
I guess not